### PR TITLE
Test different workloads (transaction types) to see TPS achieved

### PR DESCRIPTION
**Pull request by igor-aptos**

### CODE REVIEW

File: testsuite/forge-cli/src/main.rs [hunk 1]
```diff
@@ -200,9 +200,15 @@ fn main() -> Result<()> {
     logger.build();
 
     let args = Args::from_args();
-    let duration = Duration::from_secs(args.duration_secs as u64);
+    let duration = Duration::from_secs(15 * 8 * 60); // * args.duration_secs as u64);
```
Comment (sitalkedia) [inline testsuite/forge-cli/src/main.rs:203]: Please remove before landing.

File: testsuite/forge-cli/src/main.rs [hunk 2]
```diff
@@ -200,9 +200,15 @@ fn main() -> Result<()> {
...
     let suite_name: &str = args.suite.as_ref();
 
+    let suite_name = if suite_name == "compat" {
```
Comment (sitalkedia) [inline testsuite/forge-cli/src/main.rs:206]: Same as above

File: crates/transaction-emitter-lib/src/transaction_generator/mod.rs [hunk 3]
```diff
@@ -75,64 +76,82 @@ pub async fn create_txn_generator_creator(
...
+                        sender_use_account_pool,
+                    } => wrap_accounts_pool(
+                        Box::new(P2PTransactionGeneratorCreator::new(
+                            txn_factory.clone(),
+                            SEND_AMOUNT,
+                            all_addresses.clone(),
+                            *invalid_transaction_ratio,
+                        )),
+                        *sender_use_account_pool,
+                        accounts_pool.clone(),
```
Comment (sitalkedia) [inline crates/transaction-emitter-lib/src/transaction_generator/mod.rs:108]: Where do you generate this account pool? I don't see corresponding change? Also, how many accounts do we generate?

Comment (igor-aptos) [reply crates/transaction-emitter-lib/src/transaction_generator/mod.rs:108]: TransactionType::AccountGeneration has add_created_accounts_to_pool field. 
so in order to use this, you need to have two phases, one with generation and second with your transaction

**Review by msmouse — APPROVED** [review]

**Pull request merged**